scripts/us_bea/states_gdp/README.md

```diff
@@ -0,0 +1,8 @@
+# Importing US Bureau of Economic Analysis quarterly GDP data per US state
+
+## Cleaned Data
+1. [states_gdp.csv](states_gdp.csv).
+
+## Scripts
+1. [import_data.py](import_data.py)
+  1. Pulls data from the BEA API and cleans it, yielding a CSV file, e.g. [states_gdp.csv](states_gdp.csv).
```

Comment:
The sublist item has a number. Does it need one? Can the sentences just be plain text following the parent number list entry?

Also, will there be more here? a one-item numbered list isn't very useful. maybe make it a bullet point instead if it needs a list format.

Reply:
Can you also list where on BEA it gets the data, or the API call its using, or link to the API docs

Reply:
It shouldn't say API here (this is obsolete) since we are now doing direct downloads. Will change this.